src/JsonSchema/Validator.elm

```diff
@@ -89,6 +94,23 @@ validateMaxItems int values =
                 [ ( [], HasMoreItemsThan maxItems ) ]
 
 
+validateUniqueItems : Bool -> Maybe Schema -> Array.Array Value -> List Error
+validateUniqueItems unique item values =
+    if not unique then
+        []
+    else
+        List.map (\v -> Json.Encode.encode 0 v) (Array.toList values)
```

Comment:
Same comment as above with regards to key order. If two json objects are the same except for the rendering of the keys they are not unique.